grumphp.yml

```diff
@@ -1,66 +1,68 @@
 imports:
-  - { resource: vendor/drupol/php-conventions/config/php73/grumphp.yml }
+    - { resource: vendor/drupol/php-conventions/config/php73/grumphp.yml }
 
 parameters:
-  # GrumPHP License
-  tasks.license.holder: Pol Dellaiera
-  tasks.license.date_from: 2019
+    # GrumPHP License
+    tasks.license.holder: Pol Dellaiera
+    tasks.license.date_from: 2019
 
-  # PHP CS Fixer
-  tasks.phpcsfixer.config: .php-cs-fixer.dist.php
-  tasks.phpcsfixer.diff: true
+    # PHP CS Fixer
+    tasks.phpcsfixer.config: .php-cs-fixer.dist.php
+    tasks.phpcsfixer.diff: true
 
-  # PHPCS
-  tasks.phpcs.ignore_patterns:
-      - "/.github/"
-      - "/.idea/"
-      - "/build/"
-      - "/benchmarks/"
-      - "/docs/"
-      - "/node_modules/"
-      - "/resource/"
-      - "/spec/"
-      - "/tests/"
-      - "/var/"
-      - "/vendor/"
+    # PHPCS
+    tasks.phpcs.ignore_patterns:
+        - "/.github/"
+        - "/.idea/"
+        - "/build/"
+        - "/benchmarks/"
+        - "/docs/"
+        - "/node_modules/"
+        - "/resource/"
+        - "/spec/"
+        - "/tests/"
+        - "/var/"
+        - "/vendor/"
 
-  # PHPStan
-  tasks.phpstan.level: 8
-  tasks.phpstan.blocking: true
-  tasks.phpstan.ignore_patterns:
-    - "/.github/"
-    - "/.idea/"
-    - "/build/"
-    - "/benchmarks/"
-    - "/docs/"
-    - "/node_modules/"
-    - "/resource/"
-    - "/spec/"
-    - "/var/"
-    - "/vendor/"
+    # PHPStan
+    tasks.phpstan.level: 8
+    tasks.phpstan.blocking: true
+    tasks.phpstan.ignore_patterns:
+        - "/.github/"
+        - "/.idea/"
+        - "/build/"
+        - "/benchmarks/"
+        - "/docs/"
+        - "/node_modules/"
+        - "/resource/"
+        - "/spec/"
+        - "/var/"
+        - "/vendor/"
+        - "tests/unit/"
+        - "tests/src/"
 
-  # Psalm
-  tasks.psalm.blocking: true
-  tasks.psalm.ignore_patterns:
-      - "/.github/"
-      - "/.idea/"
-      - "/build/"
-      - "/benchmarks/"
-      - "/node_modules/"
-      - "/resource/"
-      - "/spec/"
-      - "/var/"
-      - "/vendor/"
+    # Psalm
+    tasks.psalm.blocking: true
+    tasks.psalm.ignore_patterns:
+        - "/.github/"
+        - "/.idea/"
+        - "/build/"
+        - "/benchmarks/"
+        - "/node_modules/"
+        - "/resource/"
+        - "/spec/"
+        - "/var/"
+        - "/vendor/"
 
-  extra_tasks:
-    phpspec:
-      verbose: true
-    clover_coverage:
-      clover_file: build/logs/clover.xml
-      level: 99
-    infection:
-      threads: 10
-      test_framework: phpspec
-      configuration: infection.json
-      min_msi: 90
-      min_covered_msi: 90
+    extra_tasks:
+        phpunit:
+            always_execute: true
+        clover_coverage:
+            clover_file: build/logs/clover.xml
+            level: 99
+        infection:
+            threads: 4
+            test_framework: phpunit
+            configuration: infection.json
+            min_msi: 90
```

Comment:
```suggestion
            min_msi: 60
```

Reply:
When the migration will be over, we'll be at least at 90%... there are many tests that I need to migrate.

It's a huge work, and I'm doing that slowly, piece by piece :)

Reply:
Sure thing! It might be easier to move this thing out of your radar until everything else is done, and then you can raise the numbers to any value you want.

Reply:
It's ok... I'll take my time, no worries about that. I guess we should come back >90% by the end of the week.

Reply:
Best of luck!

Reply:
Thanks for the help and piece of advice!

Reply:
Finally ! I'm above 90% :) except for PHP 7.4.